### ISSUE TITLE
Option to collapse branches by default

### ISSUE THREAD

**Issue by nickvanderstel** [xndcn/symbols-tree-view#7]
I think an option to have branches collapsed by default would be useful.

**Comment by grandsong** [comment]
If you unfold some items in a tree view, usually you won't be happy to see they collapse every time you save (or focus changes).

I guess your second request should be modified as "the tree won't change when files are saved".

**Comment by nickvanderstel** [comment]
Yeah, when saved or when you switch tabs. I think it would be nice if the state of the tree was maintained while a tab is open, but that really should be a separate issue.

**Comment by shanemikel** [comment]
I agree with both of these suggestions

**Comment by riker09** [comment]
I would like to have a collapsable tree as well. See #44 and #26, which address the same (or at least a similar) issue.

**Comment by dessant** [comment]
I would like to second this issue. There should be an option to collapse or expand everything by default, and the collapsed or expanded state of each class should be remembered when a file is saved, otherwise you have to look up classes and expand them every time tabs are switched and the files are automatically saved.

**Comment by sarangjo** [comment]
Hi there, I was recently added as a collaborator for this project and am trying to clean up old issues and PR's.

I like the idea of having an option to choose whether all the nested tags are open or closed by default. I think the issue of maintaining the state across saves or focus changes is a key issue to deal with, which hopefully I'll be able to fix as part of the feature merge.
